Source/pfile.cpp

```diff
@@ -316,19 +288,19 @@ void pfile_write_hero(bool writeGameData)
 
 void pfile_write_hero_demo(int demo)
 {
-	std::string savePath = GetSavePath(gSaveNumber, fmt::format("demo_{}_reference_", demo));
+	std::string savePath = GetSavePath(gSaveNumber, StrCat("demo_", demo, "_reference_"));

```

Comment:
I'm not as fond of changes like this, but I think it's acceptable (especially if there's a comment outlining the benefits over using fmt::format).